pro/src/screens/IndividualOffer/StocksEventCreation/StocksEventCreation.tsx

```diff
@@ -220,23 +98,24 @@ export const StocksEventCreation = ({
           fullContentWidth
         >
           <RecurrenceForm
-            offer={offer}
+            offerId={offer.id}
+            departmentCode={offer.venue.departmentCode}
+            priceCategories={offer.priceCategories ?? []}
             onCancel={onCancel}
-            onConfirm={onConfirm}
+            stocks={stocks}
+            setStocks={setStocks}
+            setIsOpen={setIsRecurrenceModalOpen}
           />
         </DialogBox>
       )}
 
       <ActionBar
-        isDisabled={isOfferDisabled(offer.status) || isSubmitting}
+        isDisabled={false}
```

Comment:
On ne veut plus du tout désactiver le bouton ?

Reply:
non ! 
1- c'est une mauvaise pratique de désactiver les bouton
https://adamsilver.io/blog/the-problem-with-disabled-buttons-and-what-to-do-instead/

2- le bouton ne fait presque rien ici (simplement il vérifie qu'il y a des stocks et envoit sur la page suivante)